fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1333,6 +1333,30 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long memory_tracker_interval_seconds = 60;
 
+    /**
+     * true to enable collect proc memory alloc profile
+     */
+    @ConfField(mutable = true)
+    public static boolean proc_profile_enable = true;
+
+    /**
+     * The number of seconds between proc profile collections
+     */
+    @ConfField(mutable = true)
+    public static long proc_profile_collect_interval_s = 3600;
+
+    /**
+     * The number of seconds it takes to collect single proc profile
+     */
+    @ConfField(mutable = true)
+    public static long proc_profile_collect_time_s = 600;
+
+    /**
+     * The number of days to retain profile files
+     */
+    @ConfField(mutable = true)
+    public static int proc_profile_file_retained_days = 7;
```

Comment:
2 by default?

Reply:
I think set a maximum file size makes more sense, like 1GB ?

Reply:
and also please compress the file when saving to disk

Reply:
there are two params to control the deletion of expired files. proc_profile_file_retained_days and proc_profile_file_retained_size_bytes